src/StatusQ/Components/StatusCommunityCard.qml

```diff
@@ -102,180 +130,330 @@ Rectangle {
 
     QtObject {
         id: d
-        property int dMargins: 12
+        property int cardWidth: 335
+        property int bannerHeigth: 64
+        property int cardHeigth: 190
+        property int totalHeigth: 230
+        property int margins: 12
+        property int bannerRadius: 20
```

Comment:
Wondering if this shouldn't be relative to the banner window size and synced with the banner editor to keep consistency with the banner editing. If that is the case wondering where the relative definitions should reside, `Style`, `Theme`?

Reply:
Still pending answer from the designers. It will be addressed later on here: https://github.com/status-im/StatusQ/issues/722

Reply:
@stefandunca just chatted to @noeliaSD, hopefully I understand the question correctly.  Your image cropping and scaling tool should export the image as rectangle, the image it exports shouldn't have rounded corners. This image is then used in multiple contexts in Status Mobile, and currently in one context in Status Desktop (the cards).  These different contexts where the image is used have different aspect ratios, and could hypothetically have different types of corners (square, rounded with a radius), so applying a corner radius to these images should happen in the UI code where the image is used.  

As far as I can see the aspect ratio that the image cropping and scaling is correct e.g. sufficient to meet the needs of all the different contexts.  

We could update the border radius the user sees in the image cropping and scaling tool to match the border radius used in the community cards.  But this is a low priority improvement, and anyway the actual image that the tool exports should be a rectangle that doesn't have rounded corners (because the corner rounding will happen in the UI code where the image is used if corner rounding is needed).

Does this answer your question?

Reply:
@John-44 yes, you nailed it and agree that this is a light concern and should be addressed later on with low priority and only if we have significant visual differences.